docs/guidance/rayservice.md

```diff
@@ -0,0 +1,158 @@
+
+Deploy the operator
+
+`kubectl apply -k "github.com/ray-project/kuberay/ray-operator/config/default"`
+
+Check that the controller is running.
+
+```shell
+$ kubectl get deployments -n ray-system
+NAME           READY   UP-TO-DATE   AVAILABLE   AGE
+ray-operator   1/1     1            1           40s
+
+$ kubectl get pods -n ray-system
+NAME                            READY   STATUS    RESTARTS   AGE
+ray-operator-75dbbf8587-5lrvn   1/1     Running   0          31s
+```
+
+Delete the operator
+
+`kubectl delete -k "github.com/ray-project/kuberay/ray-operator/config/default"`
+
+### Run an example cluster
+
+There is one example config file to deploy RaySerive included here:
+[ray_v1alpha1_rayservice.yaml](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml)
+
+```shell
+# Create a ray service and deploy fruit deployment graph.
+$ kubectl apply -f config/samples/ray_v1alpha1_rayservice.yaml
+```
+
+```shell
+# List running RayServices.
+$ kubectl get rayservice
+NAME                AGE
+rayservice-sample   7s
+```
+
+```shell
+# The created RayService should include a head pod, a worker pod, and four services.
+$ kubectl get pods
+NAME                                                      READY   STATUS    RESTARTS   AGE
+ervice-sample-raycluster-qd2vl-worker-small-group-bxpp6   1/1     Running   0          24m
+rayservice-sample-raycluster-qd2vl-head-45hj4             1/1     Running   0          24m
+
+$ kubectl get services
+NAME                                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                          AGE
+kubernetes                                         ClusterIP   10.100.0.1       <none>        443/TCP                                          62d
+rayservice-sample-head-svc                         ClusterIP   10.100.34.24     <none>        6379/TCP,8265/TCP,10001/TCP,8000/TCP,52365/TCP   24m
+rayservice-sample-raycluster-qd2vl-dashboard-svc   ClusterIP   10.100.109.177   <none>        52365/TCP                                        24m
+rayservice-sample-raycluster-qd2vl-head-svc        ClusterIP   10.100.180.221   <none>        6379/TCP,8265/TCP,10001/TCP,8000/TCP,52365/TCP   24m
+rayservice-sample-serve-svc                        ClusterIP   10.100.39.92     <none>        8000/TCP                                         24m
+```
+
+### Access User Service
+
+The users' traffic can go through the `serve` service (for example, `rayservice-sample-serve-svc`).
+
+Run a curl pod:
+`kubectl run curl --image=radial/busyboxplus:curl -i --tty`
+
+For the fruit example deployment, you can try the following request
+```shell
+[ root@curl:/ ]$ curl  -X POST -H 'Content-Type: application/json' rayservice-sample-serve-svc.default.svc.cluster.local:8000 -d '["MANGO", 2]'
+6
+```
+You can get the response as `6`.
+
+* Note: serve-svc will do traffic routing among all the workers which have serve deployments. It is HA in general.
```

Comment:
These notes are all about serve-svc being `HA in general`.
Maybe you could start by saying that serve-svc is HA and follow with the notes as explanation.